addons/refuel/CfgVehicles.hpp

```diff
@@ -71,35 +71,35 @@ class CfgVehicles {
                 icon = QPATHTOF(ui\icon_refuel_interact.paa);
                 class GVAR(PickUpNozzle) {
                     displayName = CSTRING(TakeNozzle);
-                    condition = QUOTE([ARR_2(_player,_target)] call FUNC(canTakeNozzle));
-                    statement = QUOTE([ARR_2(_player,_target)] call FUNC(takeNozzle));
+                    condition = QUOTE([ARR_2(_player,_target)] call DFUNC(canTakeNozzle));
+                    statement = QUOTE([ARR_2(_player,_target)] call DFUNC(takeNozzle));
```

Comment:
DFUNC should only be used when assigning code to a function, like this:
```sqf
DFUNC(foo) = compileFinal "...";
```

Reply:
I've implemented the changes you suggested. Didn't know that was the specific use case for DFUNC, thanks for clearing that up.

Reply:
> DFUNC should only be used when assigning code to a function, like this:

No, [Coding Guidelines](https://ace3.acemod.org/wiki/development/coding-guidelines.html#21-modulepbo-specific-macro-usage) object. In some situations `FUNC` can become not quotable and using of `DFUNC` is legal in configs.